youtube-client/src/components/Input.tsx

```diff
@@ -0,0 +1,38 @@
+import { TextField } from '@mui/material';
+import React, { FormEventHandler, useState } from 'react';
```

Comment:
[React imports could be omitted with appropriate setup](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports)

Reply:
{ FormEventHandler, useState } are not used